Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -594,67 +594,45 @@ private void AddDirectory(string dirName, CancellationToken cancellationToken)
             }
 
             _logger.LogDebug("Checking whether patching file content is necessary...");
-            if (IsPatchingFileContentNecessary(fileName))
+
+            _logger.LogDebug("Patching is necessary. Generating new file with patched content...");
```

Comment:
This log should be rephrased.

Reply:
I think this log is not needed. I removed it